### PR TITLE
Milestone #2 Object-Oriented Programming

### PR DESCRIPTION
**Pull request by blackpintz**
## User-Interface
Mapped out the steps needed to play the tic-tac-toe game.

### CODE REVIEW

**Comment by shshamim63** [comment]
## Status :red_circle: Changes Required :red_circle: 
___
Hi @blackpintz and @OdongAlican :wave: ,
Great initiative :clap: , almost there to complete this milestone. I just need to add a couple of messages and a loop. let's get to them.
___
### :spades: Mandatory Changes :spades: 
___
** Basic Hint
image
___
- There is no indication/puts related to if a user places an invalid move or a place that is already taken. Please add this scenario.
- Well, the game contains two possible results. Either there is a winner(I can see puts that describe the situation) or it is a draw(Maybe a comment in the code about this scenario.)
- The main purpose of the milestone is that you have all the possible scenarios on your hand before you start implementing the logic. So a loop is actually needed to describe that you are aware that are some repetitive tasks gonna be placed in here.
```ruby
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw 
    game_on = false
  end
end
```
___
#### Please fix the issues mentioned above and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/)

**Comment by blackpintz** [comment]
Made the requested changes.

File: bin/main.rb [hunk 1]
```diff
@@ -1,3 +1,116 @@
...
+puts 'Player two'
+
+player_two = gets.chomp
+
+puts "Player one is #{player_one} and player two is #{player_two}"
+
+game_on = true
+winner = false
+
+while game_on && !winner
```
Comment (whiz25) [inline bin/main.rb:16]: - [ ] is this not supposed to be `while game_on && winner`?. If there is a winner `(!winner)` don't you think we should print out the winner and not continue playing the game?

Comment (blackpintz) [reply bin/main.rb:16]: Oh yes, let me change that now. Thank you.

**Pull request merged**